src/styles/theme/_variables.scss

```diff
@@ -3,3 +3,16 @@ $navbar-nav-item-height: 60px;
 
 $directions: ("t": "top", "r": "right", "b": "bottom", "l": "left", "x": ("right", "left"), "y": ("top", "bottom"));
 $sizes: ("xs": $oc-space-xsmall, "s": $oc-space-small, "m": $oc-space-medium, "l": $oc-space-large, "xl": $oc-space-xlarge, "xxl": $oc-space-xxlarge);
+
+// these are the ones the removal of ukit complains about
+// copy them over accordingly and decide whether to add
+// to theme or not
+
+$global-control-height: 15px;
+$global-border-width: 2px;
+
+$global-control-small-height: 30px;
```

Comment:
Why is the small height double the large height?

Reply:
Dev leftover and great catch, I just pasted any values in there (see comment above)